examples/api/test/material/input_decorator/input_decoration.label.0_test.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/input_decorator/input_decoration.label.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Shows the whole app', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('Shows the whole app', (WidgetTester tester) async {
```

This title can be improved,